nodeup/pkg/model/kubectl.go

```diff
@@ -84,6 +84,7 @@ func (b *KubectlBuilder) Build(c *fi.ModelBuilderContext) error {
 				Group: s("admin"),
 			})
 
+			// @question is this required?
```

Comment:
I think so - it is the kubeconfig file with the configuration that kubecfg will read...  I guess the point is that we could use the insecure port, but eventually we want to turn off the insecure port.